climada/engine/unsequa/calc_base.py

```diff
@@ -231,12 +242,20 @@ def make_sample(self, N, sampling_method='saltelli',
             'names' : param_labels,
             'bounds' : [[0, 1]]*len(param_labels)
             }
-
+        #not sure if we want to still force users to enter a value for N in the
+        #case of the 'ff' sampler. In the meantime, trigger a warning.
+        if sampling_method == 'ff':
+            LOGGER.warning("""You are using the 'ff' sampler which does not require
+                           a value for 'N. The entered N value will be ignored
+                           in the sampling process.""")
         uniform_base_sample = self._make_uniform_base_sample(N, problem_sa,
                                                              sampling_method,
                                                              sampling_kwargs)
         df_samples = pd.DataFrame(uniform_base_sample, columns=param_labels)
         for param in list(df_samples):
+            if param not in self.distr_dict: #dummy_0 param added to uniform_base_sample
+                                             #when using ff method, need to ignore it?
+                continue
```

Comment:
I think we should rather explicitly catch the case for `dummy_` variables and only ignore these instead of all possible params.

Reply:
Do we still need this line of code? As we now raise a `ValueError` when the number of parameters is not a power of 2 and require the user to input their own dummy parameter beforehand. Then if the user inputs their own dummy parameter, the dummy parameters should be covered by the ```df_samples[param] = df_samples[param].apply(self.distr_dict[param].ppf)```, correct?